src/signer.ts

```diff
@@ -109,9 +116,24 @@ export class Signer {
    */
   disconnect = (): void => {
     window.removeEventListener('message', this.onMessageListener);
+    this.#owner = null;
     this.#walletOrigin = null;
   };
 
+  /**
+   * Sets owner to signer, thats allows to communicate with a relying party.
```

Comment:
I would reuse the existing JS Docs that describes the owner in `SignerOptionsSchema`. To some extent we just move it from the option argument to an internal variable.

```
/**
   * The owner who interacts with the signer.
   *
   * When the signer is initialized, the owner should be signed in to the consumer dApp.
   * Upon signing out, it is up to the consumer to disconnect the signer.
   */
```